test_unit/test_print.py

```diff
@@ -0,0 +1,72 @@
+'''Testing file for choosing the default filepath (root) based on user's machines'''
+import toml
+import os
+from logic.common import utils_file
+
+#--------------------------------------------------#
+'''Child Test Functions'''
+
+def test_s1():
+	'''Test the join function'''
+	list_s = ["line1", "line2"]
+	print(list_s)
+	print('\n'.join(list_s))
+#	utils_file._print_list(list_s)
+	utils_file.print_linebreak()
```

Comment:
Hmmm... I appreciate the attempt! Though, this isn't how unit tests actually go. The goal of unit tests is that if something breaks in the future due to a future update, the unit tests would catch that and inform you of the failure. So there needs to be actual "assertion" calls (that's a thing that unit tests provide) to check the results against  controlled values.

Since there are no assertion calls here, say a future update makes it so that print_linebreak() actually does nothing. Would this test catch it and inform us with a clear "FAILURE DETECTED" message? It wouldn't.

And that's the advantage of real Unit tests.

Anyway, don't worry about fixing this though, I'll lead with an official unit test check-in in the future so you have an example to work against.

Reply:
I still haven't read the Unit Tests that you linked on Trello... orz

But yeah, I can kinda see how they're meant to be used with the little YouTube skit. Regardless, it would be awesome if I'm given a quick example (as opposed to reading bunch of text)! So yeah, if it's not too much of a hassle, I would greatly appreciate some help here since I have 0 experience in this whole Unit Test.